ember-cli-rails.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = ">= 2.5.0"
 
-  spec.add_dependency "ember-cli-rails-assets", "~> 0.6.2"
+  spec.add_dependency "ember-cli-rails-assets", ">= 0.6.2", "< 0.8"
```

Comment:
What makes it important that the version is `< 0.8`?

Reply:
The reason for this is to prevent break due to dependency's breaking changes.
However, It can be more loose.
How do you think about changing it as `"< 1.0"`?

Reply:
Thanks for your reaction 👍 
Updated e015ff17ad7f4ad83c9799fa598bc9650e67e0c9